woocommerce/woocommerce-ios
# Order Details: Use navigation controller to wrap EditCustomerNote instead of NavigationView

Closes #6405 


### Description

This PR removes the extra split view for the Edit customer note flow when the Orders tab is embedded in a split view.

For some reason `NavigationView` causes EditCustomNote to be displayed in a split view. 

The solution is to wrap the EditCustomNoteHostingController within a navigation controller instead.

### Testing instructions

1. Build the app to an iPad or iPhone Pro Max in landscape mode
2. Navigate to Orders tab > select an Order > Select Edit Customer Note
3. Notice that the view is presented modally and has the correct color for the navigation bar buttons.


### Screenshots

https://user-images.githubusercontent.com/5533851/157650698-0740da55-e894-4bca-96c1-50ee617c6409.mov

---
